.github/workflows/python-tests.yml

```
@@ -22,13 +22,16 @@ on:
       - 'pytest.ini'
       - '.github/workflows/python-tests.yml'
   workflow_dispatch:
+permissions:
+  contents: read
+
 
 permissions:
```

actionlint (1.7.9)

key "permissions" is duplicated in workflow. previously defined at line:25,col:1

(syntax-check)